engine/api/migrate/migrate_workflow.go

```diff
@@ -20,10 +20,15 @@ const (
 	STATUS_DONE     = "DONE"
 )
 
-func MigrateToWorkflow(db gorp.SqlExecutor, store cache.Store, cdTree []sdk.CDPipeline, proj *sdk.Project, u *sdk.User, force bool) error {
+func MigrateToWorkflow(db gorp.SqlExecutor, store cache.Store, cdTree []sdk.CDPipeline, proj *sdk.Project, u *sdk.User, force, disablePrefix, withCurrentVersion, withRepositoryWebHook bool) ([]sdk.Workflow, error) {
```

Comment:
func name will be used as migrate.MigrateToWorkflow by other packages, and that stutters; consider calling this ToWorkflow